### DISCUSSION TITLE
.netstandard 2.0 support

### DISCUSSION THREAD

**Discussion by vkirienko** [SixLabors/Fonts#358] [Q&A] [answered]
Hi, 

Was there a real reason why .netstandard 2.0 support was dropped in 2.0.0 version? It is very unfortunate because we still using this package in .NET 4.8 (ClosedXML depends on it).

Regards,
Vlad Kirienko

**Comment by JimBobSquarePants** [comment, marked as answer]
So I can look at optimisation opportunities not available to me with the current APIs. It also reduces the maintenance burden. 

You always still have v1 which should do what you need

**Reply by vkirienko** [reply]
Thank you for clarification.

**Reply by JimBobSquarePants** [reply]
I’m curious why you said ClosedXML depends on .NET 4.8? 

It looks to me like you choose to support it.

https://github.com/ClosedXML/ClosedXML/blob/develop/ClosedXML/ClosedXML.csproj

in which case you can multi target.

**Reply by Bykiev** [reply]
@JimBobSquarePants, can you please share what is not supported in .netstandard that is supported in .net 6?

**Reply by JimBobSquarePants** [reply]
Nullability attributes, CollectionMarshall APIs, intrinsics and much, much more. netstandard is fine for simple things but lots of performance APIs are missing.

**Comment by vkirienko** [comment]
ClosedXML depends on SixLabors/Fonts and our application used .NET 4.8. So we can't use SixLabors/Fonts 2.0.

    On Friday, September 15, 2023 at 08:06:16 PM EDT, James Jackson-South ***@***.***> wrote:  
 
 


I’m curious why you said ClosedXML depends on .NET 4.8?

It looks to me like you choose to support it.

https://github.com/ClosedXML/ClosedXML/blob/develop/ClosedXML/ClosedXML.csproj

in which case you can multi target.
